lighty-core/lighty-codecs/src/main/java/io/lighty/codecs/SerializeIdentifierCodec.java

```diff
@@ -44,19 +63,61 @@ public YangInstanceIdentifier serialize(final String identifier) {
             data = data.substring(0, data.length() - 1);
         }
         final List<String> pathArgs = Arrays.asList(data.split("/"));
-        final String[] first = pathArgs.get(0).split(":");
-        final String moduleName = first[0];
-        final Optional<? extends Module> module = this.schemaContext.findModule(moduleName);
-        if (! module.isPresent()) {
-            throw new IllegalStateException("Module with name" + moduleName + " not found");
+        final String[] nameRevision_pathStart = pathArgs.get(0).split(":");
```

Comment:
Although this is clever naming of split variable, please, don't use underscores in variable names. It's against java coding conventions.